src/components/TsvList/TsvList.js

```diff
@@ -183,6 +187,8 @@ function Row({
           } else if (setCurrentCheck && selected) {
             setCurrentCheck({})
           }
+          setItemIndex(rowKey)
+          console.log(rowKey)
```

Comment:
Probably should either remove this before merge or add an explanatory message here.